_data/ru/stage3.yml

```diff
@@ -0,0 +1,253 @@
+    m1.indices.groups[&#x22;Z&#x22;][1] === 5;
+    s1.slice(...m1.indices.groups[&#x22;Z&#x22;]) === &#x22;z&#x22;;
+    // capture groups that are not matched return &#x60;undefined&#x60;:
+    const m2 = re1.exec(&#x22;xaaay&#x22;);
+    m2.indices[1] === undefined;
+    m2.indices.groups[&#x22;Z&#x22;] === undefined;
+  presented:
+    - date: Декабрь&#xA0;2019
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2019-12/december-3.md#regexp-match-indices-performance-feedback
+  has_specification: false
+  description: Индексы соответствия регулярных выражений в ECMAScript предоставляют дополнительную информацию о начальном и конечном индексах захваченных подстрок относительно начала входной строки.
+  authors:
+    - Ron Buckton
+  champions:
+    - Ron Buckton
+  tests:
+    - 'https://github.com/tc39/test262/pull/2309'
+- title: <code>Atomics.waitAsync</code>
+  id: proposal-atomics-wait-async
+  presented:
+    - date: Декабрь&#xA0;2019
+      url: >-
+        https://github.com/tc39/notes/blob/master/meetings/2019-12/december-4.md#atomicswaitasync-for-stage-3
+  has_specification: true
+  description: Предложение для добавления атомарной операции ожидания асинхронной операции в ECMAScript, прежде всего, для использования в агентах, в которых невозможна блокировка.
+  authors:
+    - Lars Hansen
+  champions:
+    - Shu-yu Guo
+    - Lars Hansen
+  tests:
+    - 'https://github.com/tc39/test262/issues/2511'
+- title: <code>.item()</code>
+  id: proposal-item-method
+  example: |-
+    function item(n) {
+    &#x9;// ToInteger() abstract op
+    &#x9;n = Math.trunc(n) || 0;
+    &#x9;// Allow negative indexing from the end
+    &#x9;if(n &#x3C; 0) n += this.length;
+    &#x9;// OOB access is guaranteed to return undefined
+    &#x9;if(n &#x3C; 0 || n &#x3E;= this.length) return undefined;
+    &#x9;// Otherwise, this is just normal property access
+    &#x9;return this[n];
+    }
+    // Other TypedArray constructors omitted for brevity.
+    for (let C of [Array, String, Uint8Array]) {
+        Object.defineProperty(C.prototype, &#x22;item&#x22;,
+                              { value: item,
+                                writable: true,
+                                enumerable: false,
+                                configurable: true });
+    }
+  presented:
+    - date: Сентябрь&#xA0;2020
+      url: item
+  has_specification: true
+  description:  Предложение по добавлению метода <code>.item()</code> во все основные перечисляемые классы (<code>Array</code>, <code>String</code>, <code>TypedArray</code>).
+  authors:
+    - Shu-yu Guo
+    - Tab Atkins
+  champions:
+    - Shu-yu Guo
+    - Tab Atkins
+  tests:
+    - 'https://github.com/tc39/test262/pull/2812'
+- title:  Import assertions
+  id: proposal-import-assertions
+  example: >-
+    new Worker(&#x22;foo.wasm&#x22;, { type: &#x22;module&#x22;, assert: { type:
+    &#x22;webassembly&#x22; } });
+  presented:
+    - date: September&#xA0;2020
```

Comment:
```suggestion
    - date: Сентябрь&#xA0;2020
```

Reply:
this pr 8 has participants, suggest open new pr. like #220

Reply:
we might finish this patch soon and be able to work of master again. I hope to merge by end of day today